benchmarks/data/linear-algebra-mvt.data

```diff
@@ -18,12 +18,29 @@
              [17,49,9,5,31,3,37,37]],
     "bitwidth": 32
   },
-  "x20": {
-    "data": [3,31,27,39,43,6,9,1],
+  "x1_int0":{
+    "data": [20,49,17,40,11,30,29,10],
```

Comment:
Why did the input data change?

Reply:
It didn't—I just flipped the order of x1 and x2 (so now they appear in lexicographical order).